src/test/clientApp/src/main/java/org/lfenergy/operatorfabric/client/config/JacksonConfig.java

```diff
@@ -0,0 +1,48 @@
+/* Copyright (c) 2021, RTE (http://www.rte-france.com)
+ * See AUTHORS.txt
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ * SPDX-License-Identifier: MPL-2.0
+ * This file is part of the OperatorFabric project.
+ */
+
+package org.lfenergy.operatorfabric.client.config;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.datatype.jdk8.Jdk8Module;
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.http.converter.json.Jackson2ObjectMapperBuilder;
+import org.lfenergy.operatorfabric.springtools.json.InstantModule;
+
+/**
+ * Json configuration
+ *
+ *
+ */
+@Configuration
+@Slf4j
+public class JacksonConfig {
```

Comment:
Again, that's not really in the scope of this PR but the fact that you created this test app shows that there might be an issue with the existing lib: id we need this so the serialization to create the body of the card is correct from the API point of view, shouldn't it be somehow bundled with the client library (or failing that, mentioned in big red letters in the docs)?
Because my understanding of the client lib is that we could tell people who want to code stuff that sends card to OpFab "here, take that lib, and if you use the card builder in it we guarantee that the generated cards will be in a format that's acceptable by the API"...
Here if they don't add this config to their app they'll get their cards rejected at least because they'll have their dates as timestamps rather than milliseconds from epoch, right?

Reply:
Not bundled with client library to avoid adding dependency to spring framework

Reply:
Created https://opfab.atlassian.net/browse/OC-1517 to address it.